### PR TITLE
fix: ctc loss + args train scripts

### PR DESCRIPTION
**Pull request by charlesmindee**
This PR fixes ctc loss (removing softmax) and img_folder arg in both training scripts.
Any feedback is welcome!

### CODE REVIEW

**Review by fg-mindee — APPROVED** [review]
Thanks for the fix!
